lib/Sequence.fz

```diff
@@ -72,7 +72,7 @@ Sequence(T type) ref is
   count => (mapSequence i32 (_ -> 1)).fold i32.sum
 
 
-  # get the first element of this Sequence, panic if Sequence is empty
+  # get the first element of this Sequence, undefined behaviour if Sequence is empty
```

Comment:
Not respecting a pre-condition is a bug in the caller. 

I think it is better to say: _Sequence must not be empty, causes precondition failure if debug is enabled._